TF_Service_dotNet/TouchFree/Connections/MessageQueues/BaseConfigurationChangeQueueHandler.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using Ultraleap.TouchFree.Library.Configuration;
+
+namespace Ultraleap.TouchFree.Library.Connections.MessageQueues
+{
+    public abstract class BaseConfigurationChangeQueueHandler : MessageQueueHandler
+    {
+        protected BaseConfigurationChangeQueueHandler(IUpdateBehaviour _updateBehaviour, IClientConnectionManager _clientMgr) : base(_updateBehaviour, _clientMgr)
+        {
+        }
+
+        /// <summary>
+        /// Checks for invalid states of the config request
+        /// </summary>
+        /// <param name="_content">The whole json content of the request</param>
+        /// <returns>Returns a response as to the validity of the _content</returns>
+        protected ResponseToClient ValidateConfigChange(string _content)
+        {
+            ResponseToClient response = new ResponseToClient(string.Empty, "Success", string.Empty, _content);
+
+            JObject contentObj = JsonConvert.DeserializeObject<JObject>(_content);
```

Comment:
I think this would be a beneficial change but I didn't want to change too much in this PR and this code was directly moved from the MessageReceiver class

Reply:
I've added a task to see if we can do any further improvements to this (though it's more of a spike)